app/controllers/admin/resources_controller.rb

```diff
@@ -0,0 +1,52 @@
+module Admin
+  class ResourcesController < ApplicationController
+    load_and_authorize_resource :conference, find_by: :short_title
+    load_and_authorize_resource :resource, only: [:show, :edit, :update, :destroy]
```

Comment:
Yes, I have changed the permissions now to exactly as said . :slightly_smiling_face: